source/features/link-to-gihub-io.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import LinkIcon from 'octicon/link.svg';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+import LinkExternalIcon from 'octicon/link-external.svg';
+
+import features from '.';
+import {getRepositoryInfo} from '../github-helpers';
+
+async function repoListInit(): Promise<void> {
+	for (const repo of select.all<HTMLAnchorElement>('a[href$=".github.io"][itemprop="name codeRepository"]')) {
+		repo.after(
+			<a
+				href={`https://${repo.textContent!.trim()}`}
+				target="_blank"
+				rel="noopener noreferrer"
+			>
+				{" "}<LinkExternalIcon/>
+			</a>
```

Comment:
Alternatively:
```suggestion
			> <LinkExternalIcon/></a>
```

Reply:
This throws a lint error `Closing tag of a multiline JSX expression must be on its own line.`
When done like this:
```jsx
<a
	href={`https://${repo.textContent!.trim()}`}
	target="_blank"
	rel="noopener noreferrer"
> <LinkExternalIcon/>
</a>
```
It looks more like a bad case of formatting than an intentional space, which is why I added the `{" "}` to make it very evident what I'm trying to achieve. Is that acceptable?